api/Makefile

```diff
@@ -11,6 +11,9 @@ POETRY_VERSION ?= 1.8.2
 
 GUNICORN_LOGGER_CLASS ?= util.logging.GunicornJsonCapableLogger
 
+SAML_REVISION ?= v1.6.0
+RBAC_REVISION ?= v0.7.0
```

Comment:
What about workflows, ldap, auth-controller?

Reply:
Their versions are specified in pyproject.toml.

The default private module versions are here just because we needed a source of truth other than GHA workflows.

Reply:
So we no longer have e.g. `inputs.flagsmith_rbac_revision` to manage? If so, 👌